.github/workflows/sonarqube.yml

```
@@ -12,16 +12,85 @@ jobs:
+          sed -i 's|filename="api/|filename="backend/api/|g' coverage.xml
+          echo "First few lines of coverage report:"
+          head -20 coverage.xml
+
       - name: SonarCloud Scan
         uses: SonarSource/sonarcloud-github-action@v2
         with:
-          projectBaseDir: .
-        env:
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
\ No newline at end of file
+          projectBaseDir: .
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)